server/views/assessments/index.njk

```diff
@@ -0,0 +1,139 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/tabs/macro.njk" import govukTabs %}
+
+{% set pageTitle = applicationName + " - " + pageHeading  %}
+{% set mainClasses = "app-container govuk-body assessments--index" %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+
+      {% set awaitingHtml %}
+      {{
+        govukTable({
+          caption: "Applications to assess",
+          captionClasses: "govuk-table__caption--m",
+          firstCellIsHeader: true,
+          head: [
+            {
+              text: "Name"
+            },
+            {
+              text: "CRN"
+            },
+            {
+              text: "Tier"
+            },
+            {
+              text: "Arrival date"
+            },
+            {
+              text: "Current location"
+            },
+            {
+              text: "Days until application due"
+            },
+            {
+              text: "Status"
+            }
+          ],
+          rows: AssessmentUtils.awaitingAssessmentTableRows(assessments.awaiting)
+        })
+      }}
+      {% endset -%}
```

Comment:
Because it's impossible to google - what's the difference between `{% endset %}` and `{% endset -%}`

Reply:
The trailing `-` stops it from including a linebreak in the generated HTML I think - I copied and pasted that bit from the GOV.UK docs 😄